frontend/pages/ManageControlsPage/OSUpdates/components/TargetSection/TargetSection.tsx

```diff
@@ -6,37 +6,48 @@ import { IConfig } from "interfaces/config";
 import SectionHeader from "components/SectionHeader";
 import Spinner from "components/Spinner";
 
-import MacOSTargetForm from "../MacOSTargetForm";
 import WindowsTargetForm from "../WindowsTargetForm";
 import PlatformTabs from "../PlatformTabs";
 import { OSUpdatesSupportedPlatform } from "../../OSUpdates";
 
 const baseClass = "os-updates-target-section";
 
 type GetDefaultFnParams = {
+  osType?: "darwin" | "ios" | "ipados";
   currentTeamId: number;
   appConfig: IConfig;
   teamConfig?: ITeamConfig;
 };
 
-const getDefaultMacOSVersion = ({
+const getDefaultOSVersion = ({
+  osType,
   currentTeamId,
   appConfig,
   teamConfig,
 }: GetDefaultFnParams) => {
-  return currentTeamId === API_NO_TEAM_ID
-    ? appConfig?.mdm.macos_updates.minimum_version ?? ""
-    : teamConfig?.mdm?.macos_updates.minimum_version ?? "";
+  const mdmData =
+    currentTeamId === API_NO_TEAM_ID ? appConfig?.mdm : teamConfig?.mdm;
+
+  if (osType === "darwin") return mdmData?.macos_updates.minimum_version ?? "";
+  if (osType === "ios") return mdmData?.ios_updates.minimum_version ?? "";
+  if (osType === "ipados") return mdmData?.ipados_updates.minimum_version ?? "";
+  return "";
 };
 
-const getDefaultMacOSDeadline = ({
+const getDefaultDeadline = ({
+  osType,
   currentTeamId,
   appConfig,
   teamConfig,
 }: GetDefaultFnParams) => {
-  return currentTeamId === API_NO_TEAM_ID
-    ? appConfig?.mdm.macos_updates.deadline || ""
-    : teamConfig?.mdm?.macos_updates.deadline || "";
+  const mdmData =
+    currentTeamId === API_NO_TEAM_ID ? appConfig?.mdm : teamConfig?.mdm;
+
+  if (osType === "darwin") return mdmData?.macos_updates.deadline ?? "";
```

Comment:
similar to above with `.deadline`

Reply:
switched to switch case

Reply:
If we can use the mapping above I think that'd be cleanest